app/model/package.scala

```diff
@@ -21,7 +21,7 @@ case class AccessibilityForm(
   name: String,
   email: String,
   isJavascript: Boolean,
-  referrer: String,
+  referrer: Option[String],
```

Comment:
looks like referrer in FeedbackForm is not optional

Reply:
For the AccessibilityController we dont need to treat referer as an Option[String] as it works a bit differently here compared to the other controllers which call their own methods but I can change it to Option[String] for consistency?